dali/base/dadfs.cpp

```diff
@@ -999,6 +999,7 @@ protected: friend class CDistributedFile;
      */
     IDistributedFile *createNew(IFileDescriptor * fdesc, bool includeports=false);
     IDistributedSuperFile *createSuperFile(const char *logicalname,IUserDescriptor *user,bool interleaved,bool ifdoesnotexist,IDistributedFileTransaction *transaction=NULL);
+    IDistributedSuperFile *createNewSuperFile(ISuperFileDescriptor *desc, IUserDescriptor *user, bool includeports=false);
```

Comment:
includeports not used, should be remove.

I noticed it's not used on createNew() either, so could be removed there too.